packages/store/src/scalarMap.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2019 Agoric, under Apache license 2.0
+
+// @ts-check
+
+import { assert, details as X, q } from '@agoric/assert';
+import { passStyleOf } from '@agoric/marshal';
+import { mustBeComparable } from '../../same-structure';
+
+const assertKey = key => {
+  harden(key); // TODO: Just a transition kludge. Remove when possible.
```

Comment:
Can we make an issue (number copied here) to remove the kludge, so we keep track of it and know why it's here and when it can be removed?

Reply:
https://github.com/Agoric/agoric-sdk/issues/3606